espnet2/asr/frontend/whisper.py

```diff
@@ -0,0 +1,158 @@
+import contextlib
+from typing import Tuple, Union
+
+import humanfriendly
+import torch
+import torch.nn.functional as F
+from typeguard import check_argument_types
+
+from espnet2.asr.frontend.abs_frontend import AbsFrontend
+
+
+class WhisperFrontend(AbsFrontend):
+    """Speech Representation Using Encoder Outputs from OpenAI's Whisper Model:
+
+    URL: https://github.com/openai/whisper
+    """
+
+    def __init__(
+        self,
+        fs: Union[int, str] = 16000,
+        n_fft: int = 400,
```

Comment:
Is it possible to get the log-mel-filterbank paramters from the pretrained whisper model. If so, it is not necessary to use this arguments. We can use the as the default values.